app/src/main/java/io/bloco/circles/ui/auth/signup/ChooseUsernameViewModel.kt

```diff
@@ -91,6 +101,7 @@ class ChooseUsernameViewModel
 
     // Inputs
     suspend fun usernamePicked(username: String) = usernameSubmitted.emit(username)
+    suspend fun skipUsername() = newAccount.send(Unit)
```

Comment:
I think these don't need to be suspend, there's blocking versions of the send/emit.